experiments/eval.py

```diff
@@ -0,0 +1,385 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import time
+from datetime import datetime
+import traceback
+from collections import deque
+import json
+
+from absl import app, flags, logging
+
+import numpy as np
+import tensorflow as tf
+
+import cv2
+import jax
+from PIL import Image
+import imageio
+
+from flax.training import checkpoints
+from jaxrl_m.vision import encoders
+from jaxrl_m.agents import agents
+from jaxrl_m.data.text_processing import text_processors
+
+# bridge_data_robot imports
+from widowx_envs.widowx_env_service import WidowXClient, WidowXStatus
+from utils import state_to_eep, stack_obs
+
+##############################################################################
+
+np.set_printoptions(suppress=True)
+
+logging.set_verbosity(logging.WARNING)
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_multi_string(
+    "checkpoint_weights_path", None, "Path to checkpoint", required=True
+)
+flags.DEFINE_multi_string(
+    "checkpoint_config_path", None, "Path to checkpoint config JSON", required=True
+)
+flags.DEFINE_string("goal_type", None, "Goal type", required=True)
+flags.DEFINE_integer("im_size", None, "Image size", required=True)
+flags.DEFINE_string("video_save_path", None, "Path to save video")
+flags.DEFINE_string("goal_image_path", None, "Path to a single goal image")  # not used by lc
+flags.DEFINE_integer("num_timesteps", 120, "num timesteps")
+flags.DEFINE_bool("blocking", False, "Use the blocking controller")
+flags.DEFINE_spaceseplist("goal_eep", [0.3, 0.0, 0.15], "Goal position")  # not used by lc
```

Comment:
Can we make `eval_lc.py` and `eval_gc.py` also have default `goal_eep` and default `initial_eep` of `[0.3, 0.0, 0.15]`? Though maybe not too important if we plan to deprecate them anyway.